src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,224 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+
+import AddToCalendar from '../../components/AddToCalendar';
+import VAFacilityLocation from '../../components/VAFacilityLocation';
+import moment from '../../lib/moment-tz';
+import {
+  getVAAppointmentLocationId,
+  getVARFacilityId,
+  isVideoAppointment,
+} from '../../services/appointment';
+import { FETCH_STATUS, PURPOSE_TEXT } from '../../utils/constants';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import * as actions from '../redux/actions';
+import { cancelAppointment } from '../redux/actions';
+import AppointmentDateTime from './cards/confirmed/AppointmentDateTime';
+import AppointmentInstructions from './cards/confirmed/AppointmentInstructions';
+
+// Only use this when we need to pass data that comes back from one of our
+// services files to one of the older api functions
+function parseFakeFHIRId(id) {
+  return id ? id.replace('var', '') : id;
+}
+
+function formatAppointmentDate(date) {
+  if (!date.isValid()) {
+    return null;
+  }
+
+  return date.format('MMMM D, YYYY');
+}
+
+function ConfirmedAppointmentDetailsPage({
+  appointmentDetails,
+  appointmentDetailsStatus,
+  facilityData,
+  fetchConfirmedAppointmentDetails,
+  confirmedStatus,
+  showCancelButton,
+}) {
+  const { id } = useParams();
+  const history = useHistory();
+
+  const appointment = appointmentDetails?.[id];
+
+  useEffect(() => {
+    const status = confirmedStatus === FETCH_STATUS.succeeded;
+
+    if (!status) {
+      history.push('/va');
```

Comment:
I think the goal of this in Narin's PR was to go back to one of the list pages when the item wasn't already fetched, so this url would need to be one of those list page routes.

Reply:
Modified to go back to the root route.